src/main/java/com/esewa/usermanagement/constants/ExceptionMessages.java

```diff
@@ -0,0 +1,8 @@
+package com.esewa.usermanagement.constants;
+
+public class ExceptionMessages {
+    public static String UserAlreadyRegisteredCode = "1000";
```

Comment:
```suggestion
    public static final String USER_ALREADY_REGISTERED_CODE = "1000";
```

Reply:
Created separate enum file to hold these constants